health-core/src/main/java/com/netflix/runtime/health/core/SimpleHealthCheckAggregator.java

```diff
@@ -36,37 +41,56 @@
 import com.netflix.runtime.health.api.HealthIndicatorCallback;
 import com.netflix.runtime.health.api.IndicatorMatcher;
 import com.netflix.runtime.health.api.IndicatorMatchers;
+import com.netflix.spectator.api.Registry;
 
 /**
  */
 public class SimpleHealthCheckAggregator implements HealthCheckAggregator {
```

Comment:
executor creates / starts a daemon thread but never releases it.  Should implement Closeable so it's at least possible to clean up.